bin/get_snos.py

```diff
@@ -0,0 +1,165 @@
+# but WITHOUT ANY WARRANTY; without even the implied warranty of
+# MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+# GNU General Public License for more details.
+
+# You should have received a copy of the GNU General Public License
+# along with this program.  If not, see <http://www.gnu.org/licenses/>.
+
+"""Getting SNOs for two configurable satellite platforms."""
+
+import sys
+from datetime import datetime, timedelta
+
+from pyorbital.orbital import Orbital
+from pyorbital.sno_utils import get_config
+from pyorbital.sno_utils import get_arc
+from pyorbital.sno_utils import get_tle
+from pyorbital.sno_utils import get_sno_point
+
+import logging
+import time
+
+t = time.time()
+LOG = logging.getLogger('snos')
+
+handler = logging.StreamHandler(sys.stderr)
+handler.setLevel(0)
+LOG.setLevel(0)
+LOG.addHandler(handler)
+
+
+def get_arguments():
+    """Get the comman line arguments required to run the script."""
+    import argparse
+
+    parser = argparse.ArgumentParser(description='Calculate SNOS between two satellite platforms')
+    parser.add_argument("-s", "--start-datetime",
+                        required=True,
+                        dest="start_datetime",
+                        type=str,
+                        default=None,
+                        help="The datetime string corresponding to the start time of when SNOS should be calculated")
+    parser.add_argument("-e", "--end-datetime",
+                        required=True,
+                        dest="end_datetime",
+                        type=str,
+                        default=None,
+                        help="The datetime string corresponding to the end time of when SNOS should be calculated")
+    parser.add_argument("-t", "--time-window",
+                        required=True,
+                        dest="time_window",
+                        type=str,
+                        default=None,
+                        help=("The time window in number of minutes - the maximum time allowed between " +
+                              "the two SNO observations"))
+    parser.add_argument("-p", "--platform-name",
+                        required=True,
+                        dest="platform_name",
+                        type=str,
+                        default=None,
+                        help="The name of the satellite platform")
+    parser.add_argument("-c", "--configfile",
+                        required=True,
+                        dest="configfile",
+                        type=str,
+                        default=None,
+                        help="The path to the configuration file")
+    parser.add_argument("-l", "--log-file", dest="log",
+                        type=str,
+                        default=None,
+                        help="The file to log to (stdout per default).")
+
+    args = parser.parse_args()
+    return args
+
+
+if __name__ == "__main__":
```

Comment:
Encapsulate all this in `main()` function, and in `if __name__ == "__main__":` block call that. Can save some time later on when hunting a bug that is caused by polluted namespace.

Reply:
Done!